src/vma/sock/sockinfo_tcp.cpp

```diff
@@ -2405,27 +2399,6 @@ int sockinfo_tcp::listen(int backlog)
 	tcp_clone_conn((struct tcp_pcb_listen*)(&m_pcb), sockinfo_tcp::clone_conn_cb);
 
 	bool success = attach_as_uc_receiver(ROLE_TCP_SERVER);
-#ifndef DEFINED_SOCKETXTREME // if not defiend
-/*TODO ALEXR
- *
- 	if (attach_as_uc_receiver(ROLE_TCP_SERVER)) {
-		si_tcp_logdbg("Fallback the connection to os");
-		setPassthrough();
-		return orig_os_api.listen(m_fd, orig_backlog);
-	}
-*/
-	if (m_rx_ring_map.size()) {
-		if (m_rx_ring_map.size() == 1) {
-			rx_ring_map_t::iterator rx_ring_iter = m_rx_ring_map.begin();
-			m_p_rx_ring = rx_ring_iter->first;
-		}
-		si_tcp_logdbg("sock state = %d success = %d", get_tcp_state(&m_pcb), success);
-		success = true;
-	} else {
-		success = false;
-	}
```

Comment:
Do we want to remove this dbg print?
Also, success parameter can be changed here. why did you remove it ?